app/controllers/citizens/shared_ownerships_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Citizens
+  class SharedOwnershipsController < BaseController
+    def show
+      @form = LegalAidApplications::SharedOwnershipForm.new(current_params)
+    end
+
+    def update
+      @form = LegalAidApplications::SharedOwnershipForm.new(shared_ownership_params.merge(model: legal_aid_application))
+
+      if @form.save
+        if @form.model.shared_ownership?
+          render plain: 'Navigate to question 1e; What percentage of your home do you own?'
+        else
+          render plain: 'Navigate to question 2a; Do you have any savings or investments'
+        end
+      else
+        render :show
+      end
+    end
+
+    private
+
+    def legal_aid_application
+      @legal_aid_application ||= LegalAidApplication.find(session[:current_application_ref])
+    end
+
+    def current_params
+      legal_aid_application.attributes.symbolize_keys.slice(:shared_ownership)
+    end
+
+    def shared_ownership_params
+      params.require(:legal_aid_application).permit(:shared_ownership)
```

Comment:
At the moment this is failing because the `form_with(` in the view needs `scope: :legal_aid_application`

Reply:
A scope has been added